pkg/eks/api.go

```diff
@@ -44,29 +44,47 @@ import (
 	"github.com/weaveworks/eksctl/pkg/az"
 	"github.com/weaveworks/eksctl/pkg/cfn/manager"
 	ekscreds "github.com/weaveworks/eksctl/pkg/credentials"
-	"github.com/weaveworks/eksctl/pkg/kubernetes"
+	iamoidc "github.com/weaveworks/eksctl/pkg/iam/oidc"
 	kubewrapper "github.com/weaveworks/eksctl/pkg/kubernetes"
+	"github.com/weaveworks/eksctl/pkg/utils/tasks"
 	"github.com/weaveworks/eksctl/pkg/version"
 )
 
-// ClusterProvider stores information about the cluster
-type ClusterProvider struct {
+// ClusterProviderImpl stores information about the cluster
```

Comment:
Please don't call them `Impl` and `...Interface`. :)

Reply:
Open to alternative names, I just couldn't think of any better ones at the time.

Reply:
Will think of something. :)